toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -49,6 +49,10 @@ using namespace std;
 #include <sys/sysctl.h> //To retrieve MAC HW infos
 #endif
 
+#ifdef LINUX
+#include <sys/sysctl.h>
```

Comment:
As I already told you earlier, `sysconf` and the two constants you use are declared in `unistd.h` (which is already included), not `sys/sysctl.h`.